app/_components/breadcrumbs/template.njk

```diff
@@ -0,0 +1,11 @@
+{% set items = params.items | eleventyNavigationBreadcrumb(params.key) %}
+<div class="govuk-breadcrumbs{%- if params.classes %} {{ params.classes }}{% endif %}">
+  <ol class="govuk-breadcrumbs__list">
+  {%- for item in items %}
+    <li class="govuk-breadcrumbs__list-item">
+      <a class="govuk-breadcrumbs__link" href="{{ item.url | url | pretty }}">{{ item.title }}</a>
```

Comment:
What's the `| pretty` doing here?

Reply:
This filter performs the following transformations:

`path/to/file.html` → `path/to/file`
`path/to/folder/index.html` → `path/to/folder/`

This is less of an issue having moved some (but not all) pages around.

11ty’s preferences is to have all URLs end with a slash – [for good reasons](https://github.com/11ty/eleventy/issues/213#issuecomment-419672043) – although this does require having file saved out as index.html, which makes me cringe a bit.

If we were to ensure all files are output like this, we could remove this filter. This is something I considered for this PR, but stopped myself from expanding it’s scope.